src/state/queries/feed.ts

```diff
@@ -243,59 +245,89 @@ export function usePinnedFeedsInfos() {
   const {hasSession} = useSession()
   const {getAgent} = useAgent()
   const {data: preferences, isLoading: isLoadingPrefs} = usePreferencesQuery()
-  const pinnedUris = preferences?.feeds?.pinned ?? []
+  const pinnedItems = preferences?.savedFeeds.filter(feed => feed.pinned) ?? []
+  const pinnedFeeds = pinnedItems.filter(feed => feed.type === 'feed')
+  const pinnedLists = pinnedItems.filter(feed => feed.type === 'list')
 
   return useQuery({
     staleTime: STALE.INFINITY,
     enabled: !isLoadingPrefs,
     queryKey: [
       pinnedFeedInfosQueryKeyRoot,
-      (hasSession ? 'authed:' : 'unauthed:') + pinnedUris.join(','),
+      (hasSession ? 'authed:' : 'unauthed:') +
+        pinnedItems.map(f => f.value).join(','),
```

Comment:
Answer: `value` is probably a better key because in principle there's nothing preventing a saved feed's `value` changing on its own. Plus we don't really _need_ the ids here at all, the received data fully depends on the URLs.